rmw_connext_cpp/src/functions.cpp

```diff
@@ -31,25 +31,25 @@
 #  pragma clang diagnostic ignored "-Wreturn-type-c-linkage"
 # endif
 #endif
-#include <ndds/ndds_cpp.h>
-#include <ndds/ndds_requestreply_cpp.h>
+#include "ndds/ndds_cpp.h"
+#include "ndds/ndds_requestreply_cpp.h"
```

Comment:
Hmm, I'm not sure I agree. Neither Boost nor Qt examples suggest to user to use quotes when including their headers and they are not "system headers". The Google style guide doesn't say one way or the other and neither does the "include what you use" tool from Google. I don't see good rationals out there for one way or the other, but this just seems like using quotes to make the linter happy.

I suppose we just forget about it.